CONTRIBUTING.md

```diff
@@ -431,7 +420,7 @@ git config blame.ignoreRevsFile .git-blame-ignore-revs
 
 ### Pull request validation workflow details
 
-Akka uses GitHub Actions to validate pull requests, which involves checking code style, run tests, check binary compatibility, etc.
+Pekko uses GitHub Actions to validate pull requests, which involves checking code style, running tests, checking binary compatibility, etc.
 
 For existing contributors, Github Actions will run without requiring any manual intervention from a core team member.
```

Comment:
@Claudenw : there is a separate effort going on to update all the folder names. I will create a new PR once that effort is done, since that would probably take some time.